lib/puppet/provider/package/cisco.rb

```diff
@@ -178,11 +184,11 @@ def install
 
   def uninstall
```

Comment:
Same comment here - looks like XR uninstall would be using yum.uninstall instead of Cisco::Yum.uninstall.